scripts/build.ps1

```diff
@@ -29,9 +29,9 @@ if (!$env:DOTNET_BUILD_VERSION) {
     $buildnumber = $commitTime.Days
     $revnumber = $commitTime.TotalSeconds
     
-    $VersionSuffix = "$buildnumber.$revnumber"
+    $VersionSuffix = "dev-$revnumber"
```

Comment:
zips are the 4-part defined just below this. File Version [vs Assembly Version] should be the same. The intent should be consistency... easy mapping of file --> Pkg || MSI || ZIP || etc.